imagination_bank/main.py

```diff
@@ -1 +1,71 @@
-pass
+import json
+def main():
+    account_name,account_balance,answer=create_account()
+    account_name,account_balance=login_user(account_name,account_balance,answer)
+    while True:
+        choice=interact_with_user()
+        if choice in ["Q","q"]:
+            exit()
+        elif choice in ["U","u"]:
+            print(f"Username is: {account_name}")
+            print()
+
+
+
+
+def create_account():
+    account_list=[]
+    print("Hello And Welcome To RealMasterChief's Bank, The Best In The World Of Online Banking.")
+    while True:
+        answer=input("Do you have an account already? [Y/N]")
+        answer=answer.upper()
+        if answer not in ["Y","N"]:
+            print("Input not recognized. Please try again.")
+            continue
+        if answer=="Y":
+            break
+        print("It's time to create an account!")
+        account_name=input("Please Enter an account name: ")
+        account_balance=0
+        with open('account.json','w') as p:
+            json.dump([[account_name,account_balance]],p)
+        print("Account was created successfully.")
+        break
+    if answer=="Y":
+        account_name="nothing"
```

Comment:
I don't like this "nothing" pattern. Let's refactor it by moving the logic into a conditional. Like this:
```
user_input = input("Do you have an account already? [Y/N] ")
if user_input == 'Y':
    account_name, account_balance = create_account()
elif user_input == 'N':
    account_name, account_balance = login_user()